CRM/Core/BAO/RecurringEntity.php

```diff
@@ -917,7 +917,7 @@ public function mapFormValuesToDB($formParams = []) {
         }
       }
       if ($formParams['repeats_by'] == 2) {
-        if (CRM_Utils_Array::value('entity_status_1', $formParams) && CRM_Utils_Array::value('entity_status_2', $formParams)) {
+        if (CRM_Utils_Array::value('entity_status_1', $formParams) && !empty($formParams['entity_status_2'])) {
           $dbParams['entity_status'] = $formParams['entity_status_1'] . " " . $formParams['entity_status_2'];
```

Comment:
Another instance where only the last one on the line was converted.

Reply:
Got it.